etplugin/metric_collector.go

```diff
@@ -85,7 +85,7 @@ func (mc *metricCollectorImpl) ValidMetrics() (map[metricSource][]string, error)
 
 		driverName, err := mc.Ethtool.GetDriverInfo(iface.Name)
 		if err != nil {
-			return nil, err
+			continue
 		}
```

Comment:
Small detail:
This change causes error message returned by GetDriverInfo() (https://github.com/lmroz/snap-plugin-collector-ethtool/blob/master/ethtool/collector.go#L58) to be never used. Consider logging the message in GetDriverInfo() or maybe remove it.